exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,6 @@
-// Start by creating a variable `message`
-
-console.log(message);
+// Start by creating a variable `message`
+
+var message = "this will log as a string.";
+var typeOfMessage = typeof message;
+
+console.log(message + " This a "+ typeOfMessage);

```

Comment:
Super minor point - best practice is usually to include a space either side of operators like +, - etc